allel/stats/diversity.py

```diff
@@ -468,6 +475,9 @@ def windowed_diversity(pos, ac, size=None, start=None, stop=None, step=None,
     # check inputs
     if not isinstance(pos, SortedIndex):
         pos = SortedIndex(pos, copy=False)
+    # check if any position is being masked
+    if np.any(np.invert(is_accessible[pos-1])):
```

Comment:
These would be more clear if they were `np.logical_not(is_accessible[pos-1])`, maybe? Since `invert` applies to integers, but `logical_not` is clear it acts on booleans?